README.md

```diff
@@ -1,79 +1,142 @@
-- This will `synchronize` the K8s Service Endpoint list, with the Nginx LB server's Upstream server list.  
-- The primary use case is for tracking the NodePort IP:Port definitions for the Nginx Ingress Controller's `nginx-ingress Service`.  
-- With the Nginx Plus Server located external to the K8s cluster, this new controller LB function would provide an alternative TCP "Load Balancer Service" for On Premises K8s clusters, which do not have access to a Cloud providers "Service Type LoadBalancer".
-- Make the solution a native Kubernetes Component, running, configured and managed with standard K8s commands.
+? {{review for embetterment}}: NKL does not require any specific domain knowledge for configuration, though you will have to understand NGINX configuration to get the most out of this solution. There is thorough documentation available about these specifics in the `docs/` directory.
 
-<br/>
+### What does NKL do?
 
-## Reference Diagram
+tl;dr:
 
-<br/>
+_**NKL is a Kubernetes controller that monitors Services and Nodes in your cluster, and then sends API calls to an external NGINX Plus server to manage NGINX Plus Upstream servers automatically.**_
 
-![NGINX LB Server](docs/media/nginxlb-nklv2.png)
+That's all well and good, but what does that mean? Well, Kubernetes clusters require some tooling to handling routing traffic from the outside world (e.g.: the Internet, corporate network, etc.) to the cluster. 
+This is typically done with a load balancer. The load balancer is responsible for routing traffic to the appropriate Kubernetes worker node which then forwards the traffic to the appropriate Service / Pod.
 
-<br/>
+If you are using a hosted web solution -- Digital Ocean, AWS, Azure, etc. -- you can use the cloud provider's load balancer service. Those services will create a load balancer for you. 
+You can use the cloud provider's API to manage the load balancer, or you can use the cloud provider's web console.
 
-## Sample Screenshots of Runtime
+However, if you checked the first box above, you are running Kubernetes on-premise and will need to manage your own load balancer. This is where NKL comes in.
 
-<br/>
+NKL itself does not perform load balancing. Instead, NKL allows you to manage resources within your cluster and have the load balancers automatically be updated to support those changes, with tooling you are most likely already using. 
 
-### Configuration with 2 Nginx LB Servers defined (HA):
+## Getting Started
 
-![NGINX LB ConfigMap](docs/media/nkl-pod-configmap.png)
+There are few bits of administrivia to get out of the way before you can start leveraging NKL for your load balancing needs.
 
-<br/>
+As noted above, NKL really shines when you have one or more Kubernetes clusters running on-premise. With this in place,
+you need to have at least one NGINX Plus host running outside your cluster (Please refer to the [Roadmap](#Roadmap) for information about other load balancer servers). 
```

Comment:
Love that you indicate potential support for a non-paid lb.  That would give me confidence.